src/benchmarklib/tpch/tpch_queries.cpp

```diff
@@ -599,20 +599,22 @@ const char* const tpch_query_13 =
  *
  * Changes:
  *  1. Random values are hardcoded
+ *  2. "revenue[STREAM_ID]" renamed to "revenue"
  *  2. dates are not supported
  *    a. use strings as data type for now
  *    b. pre-calculate date operation
+ *  3. implicit type conversions for arithmetic operations are not supported
+ *    a. changed 1 to 1.0 explicitly
+ *  4. Removed "drop view revenue[STREAM_ID];" because with it SQLiteWrapper doesn't return a result.
```

Comment:
Implementing `DROP VIEW IF EXISTS` should be easy. Otherwise, we could change the sqlite runner so that it if the last statement(s) are DDLs, their return value gets ignored. @lanice, what do you think?

Reply:
resolved. The query is split into statements using the SQLPipeline now, and `DROP VIEW` is not handed to the SQLiteWrapper. Hyrise executes it.